app/src/main/java/com/example/triptracker/model/location/popupState.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.triptracker.model.location
+
+enum class popupState {
+  DISPLAYITINERARY,
+  DISPLAYPIN,
+  PATHOVERLAY
+}
```

Comment:
Good choice to introduce states, could you please add comments to provide explanations of their respective roles ?